src/aleph/netty.clj

```diff
@@ -363,11 +387,13 @@
   []
   (DefaultChannelGroup. GlobalEventExecutor/INSTANCE))
 
+;; TODO: is this necessary? don't the netty executors check this already?
```

Comment:
I'm interested to have the answer to this question.
I always made the assumption Aleph needed to do so.

Reply:
I checked, and looks like it's necessary. 

Some of the Netty init fns put themselves on the event loop if necessary, but once a channel is active (i.e., on the ev loop), the lifecycle methods all just use asserts.

Will remove